client/header/activity-panel/panels/inbox/cards.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { Icon, box, comment, page } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Bell } from './icons/bell';
+
+const EXTENDED_TASK_LIST_ID = 'extended_task_list';
+const ORDER_PANEL_ID = 'orders-panel';
+const REVIEWS_PANEL_ID = 'reviews-panel';
+const STOCK_PANEL_ID = 'stock-panel';
+
+export const cards = [
+	{
+		name: 'thingsToDoNext',
+		icon: <Bell />,
+		title: __( 'Things to do next', 'woocommerce-admin' ),
+		/* translators: Things the user has to do */
+		content: __( 'You have %s new things to do', 'woocommerce-admin' ),
```

Comment:
We'll need to handle the singular/plural cases here.

Reply:
Good eyes! This was added in the commit e1000b4